lang/udf_cache.hh

```diff
@@ -54,11 +55,28 @@ public:
         uint64_t cache_blocks = 0;
     };
 
-    static stats& shard_stats() {
-        static thread_local stats _stats;
+private:
+    stats _stats;
+    seastar::metrics::metric_groups _metrics;
+
+public:
+    stats& shard_stats() {
         return _stats;
     }
 
+    void setup_metrics() {
+        namespace sm = seastar::metrics;
+        _metrics.add_group("user functions", {
+            sm::make_derive("udf_hits", wasm::udf_cache::shard_stats().cache_hits,
```

Comment:
Repeats "udf" and forgets to mention "cache"

Reply:
Ok